quartz-core/src/main/java/org/quartz/core/QuartzSchedulerThread.java

```diff
@@ -27,6 +27,8 @@
 import org.quartz.SchedulerException;
 import org.quartz.Trigger;
 import org.quartz.Trigger.CompletedExecutionInstruction;
+import org.quartz.impl.jdbcjobstore.JobStoreSupport;
```

Comment:
What about adding a method to the JobStore interface (which we can do with 2.3.0 - but not 2.2.x) named something like "getAquireRetryDelay()"  and the implementation on JobStoreSupport could return the value of dbRetryInterval, and the implementation on RAMJobStore could return 50 or something small, and  etc.